### PR TITLE
Allow for custom scrubbers

### PR DESCRIPTION
**Pull request by jeregrine**
This change is [Reviewable](https://reviewable.io/reviews/getsentry/raven-elixir/36)

### CODE REVIEW

File: lib/sentry/plug.ex [hunk 1]
```diff
@@ -1,30 +1,70 @@
...
     %{
       url: "#{conn.scheme}://#{conn.host}:#{conn.port}#{conn.request_path}",
       method: conn.method,
-      data: %{
-      },
+      data: handle_request_data(conn, scrubber),
       query_string: conn.query_string,
       cookies: conn.req_cookies,
-      headers: Enum.into(conn.req_headers, %{}),
+      headers: Enum.into(conn.req_headers, %{}) |> scrub_headers(),
```
Comment (mitchellhenke) [inline lib/sentry/plug.ex]: should we accept a function for scrubbing headers as well?

File: test/plug_test.exs [hunk 2]
```diff
@@ -54,4 +62,29 @@ defmodule Sentry.PlugTest do
...
   end
+
+  test "handles data scrubbing" do
+    conn = conn(:post, "/error_route", %{
+      "hello" => "world",
+      "password" => "test",
+      "cc" => "4242424242424242" })
+    |> put_req_cookie("cookie_key", "cookie_value")
+    |> put_req_header("accept-language", "en-US")
+    |> put_req_header("authorization", "ignorme")
```
Comment (mitchellhenke) [inline test/plug_test.exs]: ignore* ?

**Comment by mitchellhenke** [comment]
So with this, you don't get to send up data if you don't implement a scrubber, correct?

**Comment by jeregrine** [comment]
@mitchellhenke correct

File: test/plug_test.exs (hunk 2 above)
Comment (jeregrine) [inline test/plug_test.exs]: just a test value

File: lib/sentry/plug.ex (hunk 1 above)
Comment (jeregrine) [inline lib/sentry/plug.ex]: thought about it but its not a documented feature

**Comment by mitchellhenke** [comment]
👍

**Pull request merged**